core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -26,6 +29,7 @@
     private final ScrollOffsetCalculator scrollOffsetCalculator;
     private final FastForwarder fastForwarder;
 
+    private Set<ScrollingPageChangeListener> onScrollingPageChangeListeners;
```

Comment:
I +1 List being clearer, there is no way a dupe can appear in this list, so Set confused me